tests/playwright/controls.py

```diff
@@ -2693,31 +2725,48 @@ def expect_full_screen(
             int(open), timeout=timeout
         )
 
+    def expect_full_screen_enabled(
```

Comment:
You were right. Reverting name change

```suggestion
    def expect_full_screen_available(
```